pkgs/applications/networking/p2p/qbittorrent/default.nix

```diff
@@ -10,18 +10,24 @@ assert guiSupport -> (dbus_libs != null);
 with stdenv.lib;
 stdenv.mkDerivation rec {
   name = "qbittorrent-${version}";
-  version = "3.3.3";
+  version = "3.3.4";
 
   src = fetchurl {
     url = "mirror://sourceforge/qbittorrent/${name}.tar.xz";
-    sha256 = "0lyv230vqwb77isjqm6fwwgv8hdap88zir9yrccj0qxj7zf8p3cw";
+    sha256 = "1f4impsjck8anl39pwypsck7j6xw0dl18qd0b4xi23r45jvx9l60";
   };
 
-  nativeBuildInputs = [ pkgconfig which ];
+  nativeBuildInputs = [ pkgconfig which qt5.qmakeHook ];
```

Comment:
@simonvandel Rather than import `qt5` and add `qt5.qmakeHook` to the inputs, you should import `qmakeHook` and call the package with `qt5.callPackage` in `all-packages.nix`. This is necessary to ensure that every dependency of your package which needs Qt 5 is built with the same version.